h5p-bildetema-words-topic-image/src/utils/figure/figure.utils.test.ts

```diff
@@ -6,28 +6,22 @@ describe(renderFigure.name, () => {
   it("should create an SVG circle if two points are provided", () => {
     const pointA: Point = { x: 0, y: 0 };
     const pointB: Point = { x: 1, y: 1 };
-
-    const actual = renderFigure([pointA, pointB]);
-
-    expect(actual.startsWith("<circle")).toBeTruthy();
+    const actual = renderFigure([pointA, pointB], 0, 0);
+    expect(actual.startsWith("<ellipse")).toBeTruthy();
   });
 
   it("should create an SVG polygon if more than two points are provided", () => {
     const pointA: Point = { x: 0, y: 0 };
     const pointB: Point = { x: 1, y: 1 };
     const pointC: Point = { x: 2, y: 2 };
-
-    const actual = renderFigure([pointA, pointB, pointC]);
-
+    const actual = renderFigure([pointA, pointB, pointC], 0, 0);
     expect(actual.startsWith("<polygon")).toBeTruthy();
   });
 
   it("should return an empty string if less than two points are provided", () => {
     const pointA: Point = { x: 0, y: 0 };
-
-    const actual1 = renderFigure([pointA]);
-    const actual2 = renderFigure([]);
-
+    const actual1 = renderFigure([pointA], 0, 0);
+    const actual2 = renderFigure([], 0, 0);
     expect(actual1).toBe("");
```

Comment:
?

Reply:
`Rotations` and `ellipseRadius` are the new parameters add for `renderFigure` which causing tests to fail.

Reply:
Should we have `Rotation` and `ellipseRadius` as optional instead?

Reply:
Keeping it as it for now. We can update tests later if needed.